packages/clarity-js/src/core/task.ts

```diff
@@ -126,7 +126,8 @@ export async function suspend(timer: Timer): Promise<Task> {
     let id = key(timer);
     if (id in tracker) {
         stop(timer);
-        tracker[id].yield = (await wait()).timeRemaining();
+        // some customer polyfills for requestIdleCallback return null 
+        tracker[id].yield = (await wait())?.timeRemaining() || Setting.LongTask;
```

Comment:
dont we already have `let requestIdleCallback = window["requestIdleCallback"] || requestIdleCallbackPolyfill;`

so if requestIdleCallback returns null, dont we use the polyfill?